plugin/pkg/scheduler/api/v1/types.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1beta3"
+)
+
+// Where possible, json tags match the cli argument names.
+// Top level config objects and all values required for proper functioning are not "omitempty".  Any truly optional piece of config is allowed to be omitted.
+
+type Policy struct {
+	v1beta3.TypeMeta `json:",inline"`
+	// Predicates holds the information to configure the fit predicate functions
```

Comment:
Just a style nit (which applies throughout this file) -- there's no need to repeat the name of the field in the comment, since each comment is referring to just one field. So for example here you could say "Holds the information..." which is a little less verbose. (But thanks for writing good, thorough comments in this file!)